gazelle/js/typescript/tsconfig.go

```diff
@@ -183,12 +214,19 @@ func (c TsConfig) ExpandPaths(from, p string) []string {
 			matchedText := p[len(m.prefix) : len(p)-len(m.suffix)]
 			mappedPath := strings.Replace(originalPath, "*", matchedText, 1)
 
-			mappedPath = path.Clean(mappedPath)
-
-			possible = append(possible, path.Join(c.Paths.Rel, mappedPath))
+			possible = append(possible, path.Join(c.getRelativeExpansionIfLocal(mappedPath), c.Paths.Rel, mappedPath))
 		}
 	}
 
+	// Expand paths from baseUrl
+	// Must not to be absolute or relative to be expanded
+	// https://www.typescriptlang.org/tsconfig#baseUrl
+	if !baseurl_validation_regex.MatchString(p) && !path.IsAbs(p) {
+		possible = append(possible, path.Join(c.getRelativeExpansionIfLocal(p), c.BaseUrl, p))
+	}
+
+	BazelLog.Tracef("Found %d possible paths for %s: %v", len(possible), p, possible)
+
 	// Add 'rootDirs' as alternate directories for relative imports
```

Comment:
This same doesn't need to be done to `rootDir`s?

Reply:
Yep it should moved it down